plugins/newsdownloader.koplugin/main.lua

```diff
@@ -15,7 +15,7 @@ local ltn12 = require("ltn12")
 local NewsDownloader = WidgetContainer:new{}
 
 local initialized = false  -- for only once lazy initialization
-local FEED_CONFIG_FILE = "feeds.xml"
+local FEED_CONFIG_FILE = "feed_config.lua"
```

Comment:
File type/structure is lua, ok. 

But can it have different extension - ".conf"? 
E. G. "feed.conf", "feed_source.conf" etc, so it's explicitly seen that's user configuration file, not another developer file? 
Does lua support that?